examples/client-logger/app-insights-logger/README.md

```diff
@@ -0,0 +1,70 @@
+# @fluid-example/app-insights-logger
+
+## Overview
+
+This package provides a simple Fluid application complete with a UI view in [React](https://react.dev/) to test the Fluid App Insights telemetry logger that will route typical Fluid telemetry to configured Azure App Insights.
+
+<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE&devDependency=TRUE) -->
```

Comment:
I'm assuming consumers will generally use this as a non-dev dependency. So I would probably update to the following (and re-run generation):
```suggestion
<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE&devDependency=FALSE) -->
```